notebooks/13_Theta_phase_and_power.ipynb

```diff
@@ -0,0 +1,1123 @@
+{
```

Comment:
<p>It may be worth showing the primary keys for LFPBand in order to demonstrate what the <code>lfp_key</code> is specifying. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
i feel that at this point (when looking at notebook 13) people should know how to use primary keys to select entries from a table. So I might skip this instruction to keep the notebook relatively simple. Let me know if it's absolutely necessary to show people how to use primary keys to index LFPBand here.